go/bls.go

```diff
@@ -371,11 +371,7 @@ func AggregateSignatures(signatures []*Signature) (*Signature, error) {
 	return aggregatedSignature, nil
 }
 
-func EncodeEpochToBytes(epochIndex uint16, maximumNonSignersPlusOne uint32, aggregatedPublicKey *PublicKey, addedPublicKeys []*PublicKey) ([]byte, error) {
-	if aggregatedPublicKey == nil {
-		return nil, NilPointerError
-	}
-
+func EncodeEpochToBytes(epochIndex uint16, maximumNonSigners uint32, addedPublicKeys []*PublicKey, shouldEncodeAggregatedPublicKey bool) ([]byte, error) {
```

Comment:
Should we also make this an explicit function? `EncodeEpochToBytesWithAggregatedKey`

Reply:
Yes!